### PR TITLE
Express & React

### PR DESCRIPTION
**Pull request by blurges**
This sets up an Express server and React front end scaffolding. If you want to check that it works on your local machine look at the `README.md` in the main directory (not the one inside `frontend`).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Alex Sobieraj
```
Comment (apmasell) [inline LICENSE]: Can this be changed to Textile Museum of Canada Volunteer Assocation?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "start": "node index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blurges/textilemuseum.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/blurges/textilemuseum/issues"
```
Comment (apmasell) [inline package.json]: These should be the main repo links.

File: frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,2138 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (apmasell) [inline frontend/README.md]: If this is all boilerplate, it's probably not worth keeping.

**Comment by blurges** [comment]
Changed the copyright owner and links in `package.json` and deleted the React `README.md`

**Pull request merged**